sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client.hpp

```diff
@@ -442,13 +439,11 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
      *
      * @remark This operation requires the certificates/import permission.
      *
-     * @param name The name of the certificate.
      * @param options The options for the request.
      * @param context The context for the operation can be used for request cancellation.
      * @return Imported certificate bundle to the vault.
      */
     Azure::Response<KeyVaultCertificateWithPolicy> ImportCertificate(
-        std::string const& name,
```

Comment:
Same here... We are removing a required-param

Reply:
it's in the properties